sdk/azidentity/client_secret_credential_test.go

```diff
@@ -79,6 +85,11 @@ func TestClientSecretCredential_GetTokenSuccess(t *testing.T) {
 }
 
 func TestClientSecretCredential_GetTokenInvalidCredentials(t *testing.T) {
+	l := azcore.Log()
+	l.SetClassifications(CredentialClassification, CredentialClassificationVerbose)
```

Comment:
Just FYI this will impact other logging tests.  If it becomes a problem just add a `defer l.SetClassification()` to reset the classifications at the end of the test.